tests/e2e/pageobjects/ide/DebugView.ts

```diff
@@ -41,6 +41,16 @@ export class DebugView {
         const runDebugButtonLocator: By = By.xpath('//span[@title=\'Start Debugging\']');
 
         await this.driverHelper.waitAndClick(runDebugButtonLocator);
+
+        await this.driverHelper.getDriver().wait(async () => {
+            Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
+            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
```

Comment:
This is a thin place, may leads to "ElementNotFound" exception. Better to use existing methods from the "DriverHelper" with waiting for the element.